src/psyclone/tests/psyir/symbols/interfaces_test.py

```diff
@@ -41,10 +41,10 @@
 
 import pytest
 
-from psyclone.psyir.symbols.interfaces import SymbolInterface, \
-    AutomaticInterface, ArgumentInterface, ImportInterface, \
-    UnresolvedInterface, StaticInterface, DefaultModuleInterface, \
-    CommonBlockInterface, UnknownInterface
+from psyclone.psyir.symbols.interfaces import \
+    (AutomaticInterface, ArgumentInterface, CommonBlockInterface,
```

Comment:
Same here, you can get rid of the backslash by bringing the parenthesis up.

Reply:
Gee, you are right, I had no idea. I always had at least one term still in the previous line. Totally agreed, that's a much better solution, thanks!

Reply:
Done